frontend/src/clients/apiClient.tsx

```diff
@@ -0,0 +1,22 @@
+export interface ListResponse<T> {
+  items: T[];
+}
+export interface ConservationStatus {
```

Comment:
Line separating interfaces
```suggestion
}

export interface ConservationStatus {
```